### PR TITLE
Add stories based on code demos

### PR DESCRIPTION
**Pull request by janpaul123**
And remove redundant stories.

### CODE REVIEW

File: stories/config.js [hunk 1]
```diff
@@ -25,8 +25,12 @@ prepareForScreenshots();
 // automatically import all files ending in *.stories.js
 // $FlowFixMe - require.context seems not correctly typed.
 const req = require.context("../packages", true, /.stories.js$/);
+// $FlowFixMe - require.context seems not correctly typed.
+const reqDocs = require.context("../docs", true, /.stories.js$/);
```
Comment (jtbandes) [inline stories/config.js]: both of these should be `\.stories\.js$`

File: docs/src/jsx/allDemos.stories.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  SpheresInstancedColor,
+  SpheresSingle,
+  Text,
+  Triangles,
+};
+
+const stories = storiesOf("Worldview docs", module);
+stories.addDecorator(withScreenshot());
+
+Object.keys(allDemos).map((demoName) => {
```
Comment (jtbandes) [inline docs/src/jsx/allDemos.stories.js]: Does Storybook automatically sort these alphabetically or should we do it manually (or organize them some other way)?

File: stories/config.js (hunk 1 above)
Comment (janpaul123) [reply stories/config.js]: Ahh good catch :)

File: docs/src/jsx/allDemos.stories.js (hunk 2 above)
Comment (janpaul123) [reply docs/src/jsx/allDemos.stories.js]: It sorts automatically!

Comment (vidaaudrey) [reply docs/src/jsx/allDemos.stories.js]: The stories follows the order of `allDemos` object

Comment (janpaul123) [reply docs/src/jsx/allDemos.stories.js]: Oh yeah I think you're right, they might not do automatic sorting after all

**Pull request merged**